tools/shell.nix

```diff
@@ -0,0 +1,70 @@
+# Development shell environment for Randovania on Nix.
+#
+# This will:
+# - create the venv and install all requirements, if it doesn't already exist
+# - set up $LD_LIBRARY_PATH with the system libraries needed for RDV
+# - define $RANDOVANIA_PATH as the root of the RDV source checkout
+# - define `randovania` as a shell function that runs RDV from the source root
```

Comment:
Comment seem wrong? It doesnt define `randovania` as a shell function, it defines `rdv`. Unless theres an alias somewhere im not seeing

Reply:
Whoops, you're right! That went through a few variations before I settled on `rdv`.